src/css/main.scss

```diff
@@ -16,5 +16,8 @@
 @import "styles/experience";
```

Comment:
I noticed it today:

![image](https://user-images.githubusercontent.com/789841/47910155-358fad00-de92-11e8-84de-c55ed47e5abd.png)

Should that line have such margin? I guess yes, but please confirm

- [ ] Confirm